group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -71,7 +71,7 @@ public class GroupCoordinatorConfig {
     /** New group coordinator configs */
     public static final String NEW_GROUP_COORDINATOR_ENABLE_CONFIG = "group.coordinator.new.enable";
     public static final String NEW_GROUP_COORDINATOR_ENABLE_DOC = "Enable the new group coordinator.";
-    public static final boolean NEW_GROUP_COORDINATOR_ENABLE_DEFAULT = false;
+    public static final boolean NEW_GROUP_COORDINATOR_ENABLE_DEFAULT = true;
```

Comment:
the default value is changed to `true`, so we need to set it to `false` explicitly when the tests want to test "old" coordinator.  Could you please update the following condition?

https://github.com/apache/kafka/blob/be3e674e78092be7e1e5f3fae35d45f12639104c/core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala#L74

Reply:
I will remove all the `kraft+kip848` and that condition in a subsequent PR. We really want all the existing tests to use the new coordinator now. If someone wants to use the old coordinator for some tests, he will have to explicitly set the configuration to disable it.

Reply:
sounds good. Let ship it